src/api/tracking/withdrawals.ts

```diff
@@ -81,13 +81,13 @@ type NurseryWithdrawalResponse = {
   error?: string;
 };
 
-export const getNurseryWithdrawal = async (withdrawalId: number): Promise<NurseryWithdrawalResponse> => {
+export const getNurseryWithdrawal = async (withdrawalId: string): Promise<NurseryWithdrawalResponse> => {
```

Comment:
We have this pattern `(id: number)` in most new API.
Calling code could theoretically pass a Number(stringId) to this API to workaround this change.
Kept the i`d: number` signature because id is defined as a number in the object type from BE.